crates/walrus-service/src/client/cli/args.rs

```diff
@@ -543,7 +550,7 @@ pub enum CliCommands {
     },
     /// Administration subcommands for storage node operators.
     NodeAdmin {
-        #[arg(long, global = true)]
+        #[arg(long, global = true, required = false)]
```

Comment:
Question: How is this related to the rest of the PR?

Reply:
gobal and required = true is forbidden according to docs and triggers assert when generating completion scripts. Also when you run "walrus node-admin" you get the traceback.

Reply:
Interesting. I found the debug assert but couldn't find this in the docs. Can you point me to where this is described?

Reply:
https://kbknapp.github.io/clap-rs/clap/struct.Arg.html#method.global

Reply:
I don't know why they changed the docs but did not remove the assert.